Ports/ccache/package.sh

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env -S bash ../.port_include.sh
-port=ccache
-version=4.6
-useconfigure=true
-files="https://github.com/ccache/ccache/releases/download/v${version}/ccache-${version}.tar.gz ccache-$version.tar.gz 73a1767ac6b7c0404a1a55f761a746d338e702883c7137fbf587023062258625"
-auth_type=sha256
-depends=("zstd")
-configopts=("-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt" "-DCMAKE_BUILD_TYPE=Release" "-DREDIS_STORAGE_BACKEND=OFF" "-GNinja")
+port='ccache'
+version='4.6.3'
+useconfigure='true'
+files="https://github.com/ccache/ccache/releases/download/v${version}/ccache-${version}.tar.gz ccache-$version.tar.gz f46ba3706ad80c30d4d5874dee2bf9227a7fcd0ccaac31b51919a3053d84bd05"
+auth_type='sha256'
+depends=('zstd')
+configopts=(
+    "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+    '-DCMAKE_BUILD_TYPE=Release'
+    '-DREDIS_STORAGE_BACKEND=OFF'
+    '-GNinja'
```

Comment:
```suggestion
    "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
    "-DCMAKE_BUILD_TYPE=Release"
    "-DREDIS_STORAGE_BACKEND=OFF"
    "-GNinja"
```

Actually, this would be nice for visual consistency, but other than that it looks fine.

Reply:
Sure, so we use `'` for strings without variables but for arrays we always use `"`.

Reply:
if its at the CLI level on Linux & BSD, you _wont_ get variable substitution inside single quote (') strings - one use for _not_ making "visual consistancy" is that you cant just add variables _willy nilly_ without also modifying the _string type_ which is a _pre-processor for Humans_ (as opposed to the compiler)